api/ByteLengthQueuingStrategy.json

```diff
@@ -58,7 +58,7 @@
           "spec_url": "https://streams.spec.whatwg.org/#blqs-constructor",
           "support": {
             "chrome": {
-              "version_added": "59"
```

Comment:
This is from wiki migration in https://github.com/mdn/browser-compat-data/pull/863 so probably wrong, yep.